site/content/docs/user/configuration.md

```diff
@@ -140,7 +140,7 @@ networking:
 {{< /codeFromInline >}}
 
 ##### Dual Stack clusters
-You can run dual stack clusters using `kind` 0.11+, on kubernetes versions 1.20+.
+You can run dual stack clusters using `kind` 0.12+, on kubernetes versions 1.20+.
```

Comment:
this change is wrong, this feature was not introduced in v0.12.0

Reply:
I overlooked this , sorry,

Ben is right , is not about being semantically correct , this section talks about the dual stack feature in kind, that is 0.11 ... just revert this change to 0.11+

Reply:
Sorrry, I will revert this change later.